### ISSUE TITLE
Move data transformation to django ninja schemas

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#755]
###Description of the Tech Debt###

In the operations form (and possibly other forms), we transform the data received from django into a format rjsf can display. This transformation has caused a couple bugs (e.g. https://github.com/bcgov/cas-registration/pull/729#issuecomment-1912852671), so this ticket it about moving the transformation into a django schema instead (unnest, use resolvers to change type if needed, etc.. Then, we can put the data directly into rjsf without any manipulation on the front end.

###Tech Debt Triage#
The purpose of our technical debt triage process is to analyze technical debt to determine risk level of the technical debt and the value in tackling that technical debt.

Risk Value Scoring:

| Level  | Value |
| ------ | ----- |
| High   | 3     |
| Medium | 2     |
| Low    | 1     |

| Technical Debt - Risk Types                                                                                                   | Level | Value |
| ----------------------------------------------------------------------------------------------------------------------------- | ----- | ----- |
| Business Area Risk - Risk of business area visibility / damage to user experience                                             |       |       |
| Developer Fault Risk - How likely will this tech debt cause a future error related to coding on top of it                     |       |       |
| System Fault Risk - Risk of system errors or application downtime                                                             |       |       |
| Time Scale Risk - Compound risk effect if left alone. How much more difficult to fix or dangerous will this become over time? |       |       |
| Time Sink Risk - How much will this tech debt slow the development process down                                               |       |       |
| TOTAL SCORE:                                                                                                                  |       |       |

**Comment by andrea-williams** [comment]
@BCerki do you think we still need this ticket?

**Comment by BCerki** [comment]
> @BCerki do you think we still need this ticket?

Yes, I think so--we still have frontend `formData` conversion happening in client/app/components/routes/operations/form/Operation.tsx and this strategy caused a bug in the past

**Comment by andrea-williams** [comment]
@BCerki  to add to this ticket list of schemas that are involved in this work

**Comment by BCerki** [comment]
I've left this as a single ticket instead of creating separate tickets per file because the first bullet is just renaming a variable

**Comment by BCerki** [comment]
This ticket may no longer be relevant if the registration part 2 form is so different it doesn't use the same schemas.
